mlflow/server/js/src/components/artifact-view-components/ShowArtifactImageView.js

```diff
@@ -52,61 +63,70 @@ class ShowArtifactImageView extends Component {
     img.src = this.getSrc();
   };
 
-  render() {
+  renderGif = () => {
+    const { loading } = this.state;
+    return (
+      <React.Fragment>
+        <div style={{ display: loading ? 'block' : 'none' }}>Loading...</div>
+        <img
+          src={this.getSrc()}
+          onLoad={() => this.setState({ loading: false })}
+          style={{ height: '100%', display: loading ? 'none' : 'block' }}
```

Comment:
I need to add an alt text here to follow `jsx-a11y/alt-text`.

Reply:
Sure, are you going to do it in this PR or the lint PR?